### ISSUE TITLE
Command setm is not supported (yet)

### ISSUE THREAD

**Issue by onmeac** [saltstack/salt#22460]
Using augeas to enforce one or more key=values in multiple files (nodes) at the same time results in error:
```
Function: augeas.change
Result: False
Comment: Error: Command setm is not supported (yet)
```
state file example to enforce PEERDNS=no in any ifcfg file present:
```
ifcfg_peer_dns:
  augeas.change:
    - context: /files/etc/sysconfig/network-scripts/
    - changes:
      - setm *[label() =~ glob('ifcfg-*')] PEERDNS no
```
manually using augeas (augtool) does work:
```
augtool> setm /files/etc/sysconfig/network-scripts/*[label() =~ glob('ifcfg-*')] PEERDNS no
augtool> save
Saved 3 file(s)
```

**Comment by rallytime** [comment]
Thanks for this report @onmeac. I suspect that this just isn't implemented yet.

**Comment by wkroon** [comment]
+1
I realy need this

**Comment by rallytime** [comment]
We'd gladly accept a pull request implementing this functionality if anyone wants to take a crack at it.

**Comment by rallytime** [comment]
@onmeac Looks like this was implemented recently. Does #25705 do what you need it to?

**Comment by rallytime** [comment]
Since we haven't heard back here, I am going to close this. If more work should be done to implement this, please let us know and we'll be happy to open this issue again and take another look.

**Comment by onmeac** [comment]
My apologies, with everything going I had forgotten to test and reply to this thread.

Either I am doing something wrong or this issue is not yet resolved.

```
yum whatprovides /usr/lib/python2.7/site-packages/salt/modules/augeas_cfg.py

salt-2015.5.3-4.el7.noarch : A parallel remote execution system
Repo        : @repo_epel
Matched from:
Filename    : /usr/lib/python2.7/site-packages/salt/modules/augeas_cfg.py
```
```
rm -f /usr/lib/python2.7/site-packages/salt/modules/augeas_cfg.py
```

```
yum -y reinstall salt

Running transaction
  Installing : salt-2015.5.3-4.el7.noarch
  Verifying  : salt-2015.5.3-4.el7.noarch

Installed:
  salt.noarch 0:2015.5.3-4.el7
```

This returns nothing
```
grep setm /usr/lib/python2.7/site-packages/salt/modules/augeas_cfg.py
```

```
salt-call state.sls augeas
----------
          ID: augeas_test
    Function: augeas.change
      Result: False
     Comment: Error: Command setm is not supported (yet)
     Started: 00:39:19.519436
    Duration: 803.598 ms
     Changes:   
```

salt --versions-report
```
 Salt: 2015.5.3
 Python: 2.7.5 (default, Jun 24 2015, 00:41:19)
 Jinja2: 2.7.2
 M2Crypto: 0.21.1
 msgpack-python: 0.4.6
 msgpack-pure: Not Installed
 pycrypto: 2.6.1
 libnacl: Not Installed
 PyYAML: 3.10
 ioflo: Not Installed
 PyZMQ: 14.3.1
 RAET: Not Installed
 ZMQ: 3.2.5
 Mako: Not Installed
 Tornado: Not Installed
```

**Comment by rallytime** [comment]
@onmeac You'll need to upgrade your version of salt before the fix in question will apply. The fix was submitting to the 2015.5 branch at the end of July, but the 2015.5.3 release was tagged on July 1st. If you test this with 2015.5.5, does this work for you?
